examples/format_osmfish.py

```diff
@@ -0,0 +1,202 @@
+import functools
+import json
+import os
+import re
+from typing import Mapping, Tuple, Union
+
+import click
+import numpy as np
+from slicedimage import ImageFormat
+
+import starfish.util.try_import
+from starfish.experiment.builder import FetchedTile, TileFetcher, write_experiment_json
+from starfish.types import Coordinates, Features, Indices, Number
+
+
+# We use this to cache images across tiles.  In the case of the osmFISH data set, volumes are saved
+# together in a single file.  To avoid reopening and decoding the TIFF file, we use a single-element
+# cache that maps between file_path and the npy file.
+@functools.lru_cache(maxsize=1)
+def cached_read_fn(file_path) -> np.ndarray:
+    return np.load(file_path)
+
+
+class osmFISHTile(FetchedTile):
+
+    def __init__(
+            self,
+            file_path: str,
+            coordinates: Mapping[Union[str, Coordinates], Union[Number, Tuple[Number, Number]]],
+            z: int
+    ) -> None:
+        """Initialize a TileFetcher for osmFISH Data"""
+        self.file_path = file_path
+        self.z = z
+        self._coordinates = coordinates
+
+    @property
+    def shape(self) -> Tuple[int, ...]:
+        return self.tile_data().shape
+
+    @property
+    def coordinates(self) -> Mapping[Union[str, Coordinates], Union[Number, Tuple[Number, Number]]]:
+        """For this single field of view test experiment, we expose dummy coordinates"""
```

Comment:
?

Reply:
Oops, You can see that I was expecting to do a worse job of this PR ;-)